internal/utils/logging.go

```diff
@@ -31,6 +31,22 @@ func ConnaisseurLogLevel() logrus.Level {
 	return LogLevel(os.Getenv(constants.LogLevel))
 }
 
+func LogFormatter(logFormat string) constants.ConnaisseurLogFormat {
```

Comment:
Not really used or useful anywhere else, right? So I guess should be private func
also nit: Probably rather `logFormat` instead of `LogFormatter` because result isn't really a thing that does anything by itself